lib/adapters/REST/rest-adapter.ts

```diff
@@ -33,8 +33,9 @@ const defaultHostParameters = {
 
 export class RestAdapter implements Adapter {
   private readonly params: RestAdapterParams
+  private readonly axiosInstance: AxiosInstance
 
-  public constructor(params: RestAdapterParams) {
+  public constructor(params: RestAdapterParams, userAgent?: string | undefined) {
```

Comment:
Should we make `userAgent` part of `params`? This class & constructor is public and we might want to add more options in the future. Passing it as part of `params` might be cleaner.

Reply:
done!